### ISSUE TITLE
TypeError caused by a malformed logging message

### ISSUE THREAD

**Issue by dimaryaz** [jupyter/nbconvert#955]
Commit 35bf5a5c3b70b91612c73321127734ab8ca6b38e added the following call:
```
                self.log.error('Notebook is invalid after preprocessor {}',
                               preprocessor)
```
However, by default, only `%` escapes are supported (see the [docs](https://docs.python.org/3/library/logging.html#logging.Formatter)).

This results in: `TypeError: not all arguments converted during string formatting`.

**Comment by pandeykartikey** [comment]
I would like to work on this issue if not already resolved.

**Comment by pandeykartikey** [comment]
I am to see any of the changes which I made in the code. Can anyone help?

**Comment by kynan** [comment]
This is fixed in #1130
